tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Wcf/TaskMethodInvokerIntegration.cs

```diff
@@ -50,10 +50,11 @@ internal static CallTargetState OnMethodBegin<TTarget>(TTarget instance, object
                 return CallTargetState.GetDefault();
             }
 
-            var operationContext = WcfCommon.GetCurrentOperationContext();
+            var operationContext = WcfCommon.GetCurrentOperationContext?.Invoke();
```

Comment:
Wait, how was this working before?

Reply:
It wasn't null before 😄 this is just doing a null check which shouldn't be needed in most cases, but it's maybe a bit bold to just ignore the potential nulls when we explicitly return them 🤷‍♂️